src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -67,8 +68,10 @@ public FeedLoadResult exportTables() {
             // If we create a schema or table on one connection, then access it in a separate connection, we have no
             // guarantee that it exists when the accessing statement is executed.
             connection = dataSource.getConnection();
-            // Include the dot separator in the table prefix.
-            // This allows everything to work even when there's no prefix.
+            // Use the Postgres text load format if we're connected to that DBMS.
+            postgresText = connection.getMetaData().getDatabaseProductName().equals("PostgreSQL");
+            // Construct where clause for routes for use in filtering GTFS data if exporting from the editor.
```

Comment:
The information in the comment can be gleaned from the variable name `routesWhereClause` and the general purpose of `where` clauses. The more important thing to document is why you would want to filter, especially since the SQL string contains a magic number 2. Something like "Construct an SQL where clause that retains only routes that have been flagged for export to public GTFS". Is there a symbolic constant somewhere for that number 2?

Reply:
Good point.  There is not a symbolic constant for the number 2.  I'm not sure where it would go.  What do you think?